Gemfile.lock

```diff
@@ -193,7 +195,7 @@ GEM
     i18n (0.9.5)
       concurrent-ruby (~> 1.0)
     ice_nine (0.11.2)
-    image_processing (1.10.3)
+    image_processing (1.11.0)
```

Comment:
### [image_processing](https://github.com/janko/image_processing)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.10.3 → 1.11.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/janko/image_processing/compare/v1.10.3...v1.11.0).

These are the individual commits:
- (9a9c906) [Disable sharpening with mini magick by default (#67)](https://github.com/janko/image_processing/commit/9a9c906cde97a3e72c45ad5570696c76f2f34b8b)
- (7cd4058) [[minimagick] Add #crop with left, top, width, height argumen…](https://github.com/janko/image_processing/commit/7cd4058e14b06b7dc999a3484fb605a6f651e2c9)
- (4b229f1) [Test on libvips 8.9.0](https://github.com/janko/image_processing/commit/4b229f19c7025af88ac07bff667dfbb21de26a8e)
- (20b92de) [Update changelog and docs for turned off sharpening](https://github.com/janko/image_processing/commit/20b92de6e1d443da5166631be6ca6299889ed809)
- (fb52ea0) [[minimagick] Handle destination path without file extension](https://github.com/janko/image_processing/commit/fb52ea0611b31142b32b7d165fa40a34d0386ba7)
- (50e9f03) [Bump to 1.11.0](https://github.com/janko/image_processing/commit/50e9f03c63824ca3dae1b4a91514914b78f7bba7)


</details>